patterns/2-structured/support-as-a-group.md

```diff
@@ -0,0 +1,86 @@
+## Title
+
+Support as a Group
+
+## Patlet
+
+What happens when the team or individual that was supporting an InnerSource project is no longer able to do so?
+Form a group of interested individuals from anywhere in the company to keep the project going.
```

Comment:
In the Patlet we are looking for problem + solution in two sentences. I would try to focus the 1st sentence more on what the core fo the problem seems to be, which is that the company is not supporting the maintenance of this project.

```suggestion
A heavily used InnerSource project is loosing its maintainers and no new maintainer is assigned by the company.
By forming a group of interested individuals from anywhere in the company you are able to keep the project going.
```

Reply:
New Patlet is now:
> What happens if a team or individual no longer supports an InnerSource project?
Keep the project alive by forming a group of interested individuals.